group5HM.spec.js

```diff
@@ -0,0 +1,32 @@
+describe('ut2019', function() {
+  	const email = 'test.user@ut2019.ee';
+	const name = 'Test User';
+	const message = 'Custom message!';
+	const urls = ['https://ut2019.voog.com/contact', 'https://ut2019.voog.com/contact-1'];
+	var ok_msg = ["Thank you!","Aitäh!"];
+	var err_msg = ["Invalid input", "Vigane sisend"];
+	var x = 0; // index for correct message list
```

Comment:
Instead of using multiple arrays for urls, ok_msg and err_msg you could have a single array which contains objects. This way you would not need to use the x and y helper variables and the test would be more readable.

For example: 
` [
 	{
 		url = 'https://ut2019.voog.com/contact', 
 		ok_msg= 'Thank you', 
 		err_msg = 'Invalid input'
 	},
 	 {
 		url = 'https://ut2019.voog.com/contact-1', 
 		ok_msg= 'Aitäh!', 
 		err_msg = 'Vigane sisend'
 	}
 ]`